### PR TITLE
Tools profile developed by the bio.tools team

### PR DESCRIPTION
**Pull request by AlasdairGray**
During the Biohackathon, the bio.tools updated the tools profile to bring it into line with the minimum model of bio.tools.

Please merge this in so that we can discuss these changes as a community.

### CODE REVIEW

**Comment by AlasdairGray** [comment]
Please can someone review this pull request

File: _devSpecs/Tool.html [hunk 1]
```diff
@@ -29,44 +29,87 @@
...
-  official_type: http://semanticscience.org/resource/SIO_000097
+  version: 0.3-DRAFT
+  version_date: 20181121T214754
+  official_type: http://schema.org/SoftwareApplication
   full_example: https://github.com/BioSchemas/specifications/tree/master/Tool/examples/
 mapping:
-- property: alternateName
+- property: additionalType
   expected_types:
   - Text
```
Comment (ljgarcia) [inline _devSpecs/Tool.html]: As terms from a controlled vocabulary should be used, would it not be better to have DefinedTerm as expected range?

File: _devSpecs/Tool.html [hunk 2]
```diff
@@ -29,44 +29,87 @@
...
-      "@type": ["SoftwareApplication", "Tool"],
-      "alternateName": "Protein feature viewer"
+      "@context": "http://schema.org",
+      "@type": "SoftwareApplication",
+      "applicationCategory": "Computational science tool"
     }
-- property: applicationCategory
+- property: applicationSubCategory
   expected_types:
   - Text
```
Comment (ljgarcia) [inline _devSpecs/Tool.html]: Same as previous comment, is not DefinedTerm better as range in this case?

File: _devSpecs/Tool.html [hunk 3]
```diff
@@ -75,96 +118,120 @@
...
-    "@type": ["SoftwareApplication", "Tool"],
-    "citation": "https://dx.doi.org/10.1016/S0022-2836(05)80360-2"
+      "@context": "http://schema.org",
+      "@type": "SoftwareApplication",
+      "citation": "https://doi.org/10.1038/nbt.3109"
     }
-- property: dateCreated
+
+    TODO 2nd EXAMPLE w/o URL
+- property: codeRepository
```
Comment (ljgarcia) [inline _devSpecs/Tool.html]: Please see https://schema.org/SoftwareSourceCode which could also be used here

File: _devSpecs/Tool.html [hunk 4]
```diff
@@ -75,96 +118,120 @@
...
   example: |-
     {
-      "@type": ["SoftwareApplication", "Tool"],
+       "@context": "http://schema.org",  "@type": "SoftwareApplication",
       "downloadURL": "https://github.com/sacdallago/cellmap"
     }
 - property: featureList
   expected_types:
-  - Text
   - URL
```
Comment (ljgarcia) [inline _devSpecs/Tool.html]: Would DefinedTerm work here?

File: _devSpecs/Tool.html [hunk 5]
```diff
@@ -173,8 +240,7 @@
     Inverse property: isPartOf.
   type: ""
   type_url: ""
-  bsc_description: |
-    Used to describe tools included into suites/bundles/workflows.
+  bsc_description: TODO
```
Comment (ljgarcia) [inline _devSpecs/Tool.html]: so the suite/bundles/workflows would be a https://bib.schema.org/Collection?

File: _devSpecs/Tool.html [hunk 6]
```diff
@@ -191,218 +257,1156 @@
...
-      "identifier": "OMICS_08141"
+      "@context": "http://schema.org",
+      "@type": "SoftwareApplication",
+      "@id": "https://bio.tools/peptideshaker",
+      "identifier": "https://bio.tools/peptideshaker"
     }
-- property: input
+- property: inputData
   expected_types:
   - URL
```
Comment (ljgarcia) [inline _devSpecs/Tool.html]: DefinedTerm?

File: _devSpecs/Tool.html [hunk 7]
```diff
@@ -191,218 +257,1156 @@
...
     {
-    "@type": ["SoftwareApplication", "Tool"],
-    "input": ["http://edamontology.org/data_2977", "http://edamontology.org/data_2976"]
+      "@context": "http://schema.org",
+      "@type": "SoftwareApplication",
+      "inputData": "http://edamontology.org/data_0945"
     }
+- property: inputFormat
+  expected_types:
+  - URL
```
Comment (ljgarcia) [inline _devSpecs/Tool.html]: DefinedTerm?

File: _devSpecs/Tool.html [hunk 8]
```diff
@@ -191,218 +257,1156 @@
...
   example: |-
     {
-      "@type": ["SoftwareApplication", "Tool"],
-      "name": "Pladipus"
+       "@context": "http://schema.org",  "@type": "SoftwareApplication",
+      "name": "PeptideShaker"
     }
 - property: operatingSystem
   expected_types:
   - Text
```
Comment (ljgarcia) [inline _devSpecs/Tool.html]: DefinedTerm?

File: _devSpecs/Tool.html [hunk 9]
```diff
@@ -191,218 +257,1156 @@
...
-      "@type": ["SoftwareApplication", "Tool"],
-      "operatingSystem": ["Windows 7 or above", "Linux/Unix", "Mac/Apple"]
+      "@context": "http://schema.org",
+      "@type": "SoftwareApplication",
+      "operatingSystem": ["Linux", "Windows", "Mac"]
     }
-- property: output
+- property: outputData
   expected_types:
   - URL
```
Comment (ljgarcia) [inline _devSpecs/Tool.html]: DefinedTerm?

File: _devSpecs/Tool.html [hunk 10]
```diff
@@ -191,218 +257,1156 @@
...
-    package, but required to run the application (Examples: DirectX, Java or .NET
-    runtime). Supersedes requirements.'
+  description: Version of the software instance.
   type: ""
   type_url: ""
-  bsc_description: ""
-  marginality: Optional
-  cardinality: MANY
+  bsc_description: Version(s) of the tool, which this information is valid for. Can
+    also be a comma-delimited list and include hyphen-separated ranges of versions.
```
Comment (ljgarcia) [inline _devSpecs/Tool.html]: If more than one, it should be a  ...

**Comment by AlasdairGray** [comment]
@ljgarcia as this is an (semi) autogenerated page from the crosswalk spreadsheet and not really looked at by the developers of the tool profile, it might make sense for us to approve this draft version and put the issues you raise as GitHub issues.

**Comment by ljgarcia** [comment]
@AlasdairGray I understand your point so I will approve

**Comment by AlasdairGray** [comment]
Did you create an issue for the points that you raised?

**Pull request merged**